doc/source/whatsnew/v0.21.0.txt

```diff
@@ -158,6 +158,62 @@ We have updated our minimum supported versions of dependencies (:issue:`15206`,
    | Bottleneck   | 1.0.0           |          |
    +--------------+-----------------+----------+
 
+.. _whatsnew_0210.api_breaking.loc:
+
+.loc with a list-like containing messing keys is Deprecated
```

Comment:
In general about this whatsnew section: I think we need to mention getitem as well (`s[[]]` or `df[[]]`)